### ISSUE TITLE
Merge an upstream fix

### ISSUE THREAD

**Issue by aunetx** [PRATAP-KUMAR/Control_Blur_Effect_On_Lock_Screen#3]
Hello again,

@fmuellner noticed on gitlab a bug in gnome shell looking at your extension, so he fixed it and merged it

If you want to fix your extension accordingly, here is the pull request: [1341] (https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1341) :)

**Comment by PRATAP-KUMAR** [comment]
Hi, I have gone through the link (pull request: 1341) is that pull request different than pull requests in github repository pull requests?

I think when gnome-shell officially updates and then If there is a problem then we should fix the `Control_Blur_Effect_On_Lock_Screen`..

Thanks for your support..

**Comment by aunetx** [comment]
Yeah it is different (I mean it is a gitlab pull request), but there are not so many changes so I guess we can adapt easily to it :)

Of course, that's better to wait for the 3.38 release; I will do a pull request on your repo when it is time if you are ok ^^

And thanks for the extension!

**Comment by PRATAP-KUMAR** [comment]
your comments are matching my view.. so go ahead when ever you feel its time to..

thank you!!!

**Comment by PRATAP-KUMAR** [comment]
Hi @aunetx please see this https://github.com/PRATAP-KUMAR/Control_Blur_Effect_On_Lock_Screen/issues/4
